src/components/pages/InstructorHome/InstructorCalender.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import { Calendar } from 'antd';
+import React, { useState } from 'react';
+import { Calendar, Badge } from 'antd';
 import './index.css';
-const InstrcutorCalender = () => {
-  function onPanelChange(value, mode) {
-    console.log(value, mode);
+const InstructorCalender = ({ dummyData }) => {
+  const [courses, setCourses] = useState(dummyData);
+  const [course, setCourse] = useState();
+
+  function getListData(value) {
+    courses.map(list => {
+      setCourse(list);
+    });
+    let listData;
+    switch (value.date()) {
+      case 8:
+        listData = [{ type: 'success', ...course }];
+        break;
+      case 10:
+        listData = [{ type: 'success', ...course }];
+        break;
+      case 15:
```

Comment:
I couldn't figure out what those case numbers mean. Looks like if we do it this way,  we will have courses on 8th, 10th and 15th of every month. Is there anyway we can add course to the calendar based on dates, like 10-10-2010?

Reply:
Good to know. Thanks.